docs/datasets_and_tables/transitdatabase.md

```diff
@@ -33,59 +33,92 @@ The data in the Airtable Transit Database is distinct but related to the followi
+| `tasks`<br><br> *Primary Field*: `Name` | Each record defines an action we are either undertaking or tracking to meet our OKRs and quarterly milestones.  These tasks are helpful for crosswalking "what we are doing" with each GTFS-entity. |
 | `gtfs datasets`<br><br>*Primary Field*: `gtfs_dataset_id` | Each record represents a gtfs dataset (feed) that is either a type of GTFS Schedule, Trip Updates, Vehicle Locations or Alerts.  A gtfs dataset MAY:<ul><li>be *disaggregated into* one or more `gtfs service data` records.</li><li>be *produced* by one or more `organizations`</li><li>be *published* by an `organizations`. |
-| `contracts`<br><br> *Primary Field*: `Name` | Each record is a contract between `organizations` to either provide one or more `products` or operate one or more `services`.  Each contract has propreties such as execution date, expiration, and renewals.  |
-| `gtfs service data` <br><br> *Primary Field*: `Name`, a combination of a set of `Services` record(s) and a single `gtfs datasets` record. | Each record links together a single `gtfs dataset` and one (if possible) or more `services`.  Additional fields define how to isolate the service within the `gtfs dataset`.  <br><br>Many services have more than one GTFS dataset which describes their service. Often these are either precursors to *final* datasets (e.g. AC Transit's GTFS dataset is a precursor to the Bay Area 511 dataset) or artifacts produced in other processes such as creating GTFS Realtime (e.g. [the VCTC GTFS produced by GMV](https://airtable.com/appPnJWrQ7ui4UmIl/tblnVt5FZ2FZmDjDx/viw3NtcDP3Qm0BYyG/recJhrNj21mYVETJG?blocks=hide)). The property `Category` indicates if this is the `primary` dataset, a `precursor` or `unknown` in order to distinguish which should be used. |
+| `gtfs service data` <br><br> *Primary Field*: `Name`, a combination a single `Services` record and a single `gtfs datasets` record. | Each record links together a single `gtfs dataset` and one (if possible) or more `services`.  Additional fields define how to isolate the service within the `gtfs dataset`.  <br><br>Many services have more than one GTFS dataset which describes their service. Often these are either precursors to *final* datasets (e.g. AC Transit's GTFS dataset is a precursor to the Bay Area 511 dataset) or artifacts produced in other processes such as creating GTFS Realtime (e.g. [the VCTC GTFS produced by GMV](https://airtable.com/appPnJWrQ7ui4UmIl/tblnVt5FZ2FZmDjDx/viw3NtcDP3Qm0BYyG/recJhrNj21mYVETJG?blocks=hide)). The property `Category` indicates if this is the `primary` dataset, a `precursor` or `unknown` in order to distinguish which should be used. See |
```

Comment:
There is an incomplete sentence at the very end:

`See  |`

Reply:
fixed in latest